devops/04 - bash/README.md

```diff
@@ -0,0 +1,57 @@
+## Prerequsite
+* Docker on your local machine to launch testing environment
+
+## Questions
+* Mention the advantages and disadvantages of bash scripts
+* What types of variables are used in bash?
+* What is pipes in shell script and how to use it?
+* How to show unique values of the second column from the CSV comma-separated file and sort results with the alphabet?
+
+## Task
+Create script which provide aggregated information about analysises and they datasets. Information about each analysis/deployment could be retrieved from `awscli.sh`:
```

Comment:
I believe that the aggregation condition should be defined better, could the multiple datasets be the same? how it should be aggregated in this case?